xla/debug_options_flags.cc

```diff
@@ -1783,6 +1784,16 @@ void MakeDebugOptionsFlags(std::vector<tsl::Flag>* flag_list,
       "Einsums that have partitioned operand(can be either LHS or RHS) that's "
       "larger than this threshold will be transformed to use windowed einsums."
       "Default is 100000"));
+  flag_list->push_back(tsl::Flag(
+      "xla_gpu_total_bytes_threshold_for_windowed_einsum",
```

Comment:
Replace "total" with "operand" in the flag name, since you don't also sum the size of the output: "xla_gpu_operand_bytes_threshold_for_windowed_einsum".

Reply:
sure, i will change it